libevm/JitVM.cpp

```diff
@@ -127,25 +127,19 @@ void getTxContext(evm_tx_context* result, evm_env* _opaqueEnv) noexcept
 
 void getBlockHash(evm_uint256be* o_hash, evm_env* _envPtr, int64_t _number)
 {
-	auto &env = *reinterpret_cast<ExtVMFace*>(_envPtr);
+	auto &env = reinterpret_cast<ExtVMFace&>(*_envPtr);
 	*o_hash = toEvmC(env.blockHash(_number));
 }
 
-int64_t call(
-	evm_env* _opaqueEnv,
-	evm_message const* _msg,
-	uint8_t* _outputData,
-	size_t _outputSize
-) noexcept
+void call(evm_result* o_result, evm_env* _opaqueEnv, evm_message const* _msg) noexcept
```

Comment:
This function is now over 70 lines long, and doing two very distinct tasks, performing create and call. Consider splitting it into at least 2 internal functions